CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+## [0.1.2] - Pre-release
+
+### Changed
+
+- Added SAMT file icon
+- React to new SAMT.yaml file
```

Comment:
"Start extension if samt.yaml exists in workspace" would be more accurate

Reply:
I like your comment more, so I took it